inst/Upgrades.qml

```diff
@@ -117,6 +117,29 @@ Upgrades
 		ChangeRename { from: "residualsColumn";								to:	"residualsSavedToDataColumn"				}
 		ChangeRename { from: "residualSdsColumn";							to:	"residualSdsSavedToDataColumn"				}
 
+		ChangeJS
+		{
+			name:		"priorRegressionCoefficients"
+			jsFunction:	function(options)
+			{
+				switch(options["priorRegressionCoefficients"])
+				{
+					case "AIC":									return "aic";
+					case "BIC":									return "bic";
+					case "EB-global":							return "ebGlobal";
+					case "EB-local":							return "ebLocal";
+					case "g-prior":								return "gPrior";
+					case "hyper-g":								return "hyperG";
+					case "hyper-g-laplace""":					return "hyperGLaplace";
+					case "hyper-g-n":							return "hyperGN";
+					case "JZS":									return "jzs";
+				}
+			}
```

Comment:
This option does not get upgraded... I am starting to see a pattern here (https://github.com/jasp-stats/jaspBsts/pull/20#discussion_r938797655). Am I going crazy and there is some obvious mistake or this deserves an issue, @JorisGoosen?
[bayesian-regression_0.16.3.jasp.zip](https://github.com/jasp-stats/jaspRegression/files/9269093/bayesian-regression_0.16.3.jasp.zip)

Reply:
There is an obvious mistake:
```js
 					case "hyper-g-laplace""":					return "hyperGLaplace"; 
```
there should not be `"""` in `"hyper-g-laplace"""`.

@JorisGoosen the current behavior of the upgrader is to ignore the option without any error when this happens and I guess that goes for any error encountered in `jsFunction`s. Perhaps something to change alongside https://github.com/jasp-stats/jaspBsts/pull/20#discussion_r939611518?

Reply:
I responded there